language_detector/main.py

```diff
@@ -1,8 +1,26 @@
 # -*- coding: utf-8 -*-
-
-"""This is the entry point of the program."""
+from .languages import LANGUAGES
 
 def detect_language(text, languages):
     """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    clean_list = text.split()
+    
+    match = {
```

Comment:
I think this could be an empty dict since we can pass more languages to the function if we wanted.